tests/test_generate_mbz_toc.py

```diff
@@ -34,7 +36,7 @@ def test_toc_creation_single_page(
         with open((html_path / html_filename), 'r') as f:
             assert f.read() == page_html
 
-    with open(md_filepath, 'r') as f:
+    with open(f'{md_filepath}/toc.md', 'r') as f:
```

Comment:
I'd prefer that we incorporate the new flag keeping the same CLI interface where the filename is passed by the user. If we go that route we'll need to revert these test changes.